tests/demo.c

```diff
@@ -27,16 +27,13 @@
 
 /*
  * Some useful constants defined by the UAVCAN specification.
- * Data type signature values can be easily obtained with the script show_data_type_info.py
  */
-#define UAVCAN_NODE_ID_ALLOCATION_DATA_TYPE_ID                      1
-#define UAVCAN_NODE_ID_ALLOCATION_DATA_TYPE_SIGNATURE               0x0b2a812620a11d40
+#define UAVCAN_NODE_ID_ALLOCATION_FIXED_SUBJECT_ID                  1
```

Comment:
None of the types used here are valid for UAVCAN v1.0. See:

- https://github.com/UAVCAN/dsdl/blob/uavcan-v1.0/uavcan/node/32085.Heartbeat.1.0.uavcan
- https://github.com/UAVCAN/dsdl/blob/uavcan-v1.0/uavcan/node/430.GetInfo.0.1.uavcan
- https://github.com/UAVCAN/dsdl/blob/uavcan-v1.0/uavcan/pnp/32742.NodeIDAllocationDataMTU8.0.1.uavcan (assuming CAN 2.0)

Should we update them now, or do you want to carry this change over into another PR?

Reply:
I have created #107 to update the demo application to use v1.0. For now, I just updated the DATA_TYPE_ID to FIXED_SUBJECT_ID to conform with the other changes done in the PR.